form-validation/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title></title>
+	<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.3/css/all.css" integrity="sha384-SZXxX4whJ79/gErwcOYf+zWLeJdY/qpuqC4cAa9rOGUstPomtqpuNWT9wdPEn2fk" crossorigin="anonymous">
+	<link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+	<div class="container">
+		<div class="header">
+			<h2>CREATE ACCOUNT</h2>
+		</div>
+		<form class="form" id="form">
+			<div class="form-control ">
+				<label >Username</label>
+			<input type="text" placeholder="beautyndbeast" id="username">
+			<i class="fas fa-check-circle"></i>
+			<i class="fas fa-exclamation-circle"></i>
+			<small>Error Message</small>
+			</div>
+			<div class="form-control ">
+				<label >Email</label>
+			<input type="email" placeholder="dead@gmail.com" id="email">
+			<i class="fas fa-check-circle"></i>
+			<i class="fas fa-exclamation-circle"></i>
+			<small>Error Message</small>
+			</div>
+			<div class="form-control">
+				<label >Password</label>
+			<input type="password" placeholder="password nahi batate " id="password">
+			<i class="fas fa-check-circle"></i>
+			<i class="fas fa-exclamation-circle"></i>
+			<small>Error Message</small>
+			</div>
+			<div class="form-control">
+				<label >Confirm Password</label>
+			<input type="password" placeholder="yeh bhi nahi batate " id="confirm">
+			<i class="fas fa-check-circle"></i>
+			<i class="fas fa-exclamation-circle"></i>
+			<small>Error Message</small>
+			</div>
+			<button>Submit</button>
+		</form>
+	</div>
+	<script type="text/javascript" src="script.js"></script>
+
+</body>
+</html>
```

Comment:
Add a new line at the end of the file.

Reply:
Use the link to increase the readability.

HTML - https://codebeautify.org/htmlviewer